### ISSUE TITLE
R CMD check failures for me

### ISSUE THREAD

**Issue by hadley** [trevorld/optparse#16]
```
checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.
```
```
checking re-building of vignette outputs ... NOTE
Error in re-building vignettes:
  ...
sh: rst2pdf: command not found
Error: processing vignette 'optparse.Rrst' failed with diagnostics:
conversion by rst2pdf failed!
Execution halted

```

**Comment by trevorld** [comment]
Thanks for the note.  The period at the end of the description title had already been fixed in the development version which I've just pushed to CRAN.  If you wish to no longer see the rst2pdf failing to build the vignette note in the future you'll need to install the knitr's system requirement `rst2pdf` which should be as easy as a:

    sudo pip install rst2pdf

Although in the past maybe it was trickier on a mac? (https://macyves.wordpress.com/2014/04/01/rst2pdf-and-rst2html5-on-os-x-10-9-2-with-xcode-5-1/).  The VignetteBuilder knitr`s dependency on the system tool `rst2pdf` is documented in my README (and knitr is also listed as a suggested package in the DESCRIPTION).  I've just added a note to the README on how to use `pip` to install rst2pdf.
